x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/page.tsx

```diff
@@ -130,6 +135,7 @@ export const Page: FC = () => {
           </EuiPageContent>
         </EuiPageBody>
       </EuiPage>
+      <HelpMenu docLink={helpLink} />
```

Comment:
I think it may be better if we add the `HelpMenu` link into all the ML pages in one single PR. Otherwise if you switch to a page where it isn't explicitly set like this, it will remain pointing to the last configured link. For example, here on the Data Visualizer page the link will open up https://www.elastic.co/guide/en/machine-learning/master/ml-dfanalytics.html as the Data Visualizer page doesn't contain its own `HelpMenu` tag.

https://www.elastic.co/guide/en/machine-learning/master/ml-dfanalytics.html

From a quick look, I think you need to set it on all the pages which contain `NavigationMenu`, which would be:

```
./access_denied/page.tsx
./datavisualizer/datavisualizer_selector.tsx
./datavisualizer/file_based/file_datavisualizer.tsx
./datavisualizer/index_based/page.tsx
./data_frame_analytics/pages/analytics_exploration/page.tsx
./data_frame_analytics/pages/analytics_management/page.tsx
./explorer/explorer.js
./jobs/jobs_list/jobs.tsx
./overview/overview_page.tsx
./settings/calendars/edit/new_calendar.js
./settings/calendars/list/calendars_list.js
./settings/filter_lists/edit/edit_filter_list.js
./settings/filter_lists/list/filter_lists.js
./settings/settings.tsx
./timeseriesexplorer/timeseriesexplorer_page.tsx
```